lib/fluent/command/plugin_generator.rb

```diff
@@ -281,7 +281,8 @@ def initialize
       @text = ""
       @preamble_source = ""
       @preamble = nil
-      URI.open(LICENSE_URL) do |io|
+      uri = URI.parse(LICENSE_URL)
```

Comment:
[this change](https://github.com/fluent/fluentd/pull/2753/files#diff-2196d7a0d4430a38196859d66653a7a0L284) breaks tests in ruby 2.4 due to `NoMethodError`(`open' called for URI:Module' ). however I couldn't get the error message since stdout was captured .... https://github.com/fluent/fluentd/blob/8610cd7f198208e2650171bedbe4c5e0769ba97d/test/command/test_plugin_generator.rb#L102